docs/content/Getting-Started/user-quickstart.md

```diff
@@ -0,0 +1,80 @@
+
+[![QuickStart test]({{config.repo_url}}/actions/workflows/docs-ecutable-qs.yml/badge.svg?branch={{config.ks_branch}})]({{config.repo_url}}/actions/workflows/docs-ecutable-qs.yml)&nbsp;&nbsp;&nbsp;
+
+
+!!! tip "Estimated time to complete this example:" 
+    ~4 minutes (after installing prerequisites)
+
+### How to deploy and use KubeStellar
```

Comment:
This heading should be less nested than the headings for the sections under it.

Reply:
agreed